root/home.tx

```diff
@@ -37,5 +37,6 @@
         <button class="btn search-btn help-btn">?</button>
     </div>
   </form>
+  <p class="helper_blurb">Just getting started with perl? Try these <a target="_blank" rel="noopener" href="https://www.perl.org/learn.html">free learning resources</a> from the Perl community</p>
```

Comment:
> Just getting started with perl?

Should that be `Perl` rather than `perl`? We're referring to the language rather than the interpreter here.

Reply:
Updated to Perl